WalletWasabi/WabiSabi/Client/AmountDecomposer.cs

```diff
@@ -496,7 +499,7 @@ private IEnumerable<Output> BreakDown(Money coinInputEffectiveValue, IEnumerable
 
 		foreach (var denom in denominations)
 		{
-			if (denom.Amount < MinAllowedOutputAmount || remaining < MinAllowedOutputAmount + ChangeFee)
```

Comment:
Here's an important part (and the following change.) Again, since `MinAllowedOutputAmount` includes now the fees, we need not to worry about the removal of `ChangeFee`. How about the modification of the `MinAllowedOutputAmount`? This'll result in filtering out denominations those would be uneconomical to create in the first place. This is almost the entire point of this PR.